docker-compose.yml

```diff
@@ -29,9 +42,27 @@ services:
     command:
       - -VV
       - --buildkitd=tcp://buildkitd:1234
+      - --db=postgres://sindri:${POSTGRES_PASSWORD}@postgres:5432/sindri?sslmode=disable
```

Comment:
The use of ${POSTGRES_PASSWORD} here should match the POSTGRES_PASSWORD in the postgres container above. If you want it to be overridable with a sane default, try this syntax: ${POSTGRES_PASSWORD:-changeme}